src/main/java/io/anserini/search/SearchCollection.java

```diff
@@ -206,7 +193,7 @@ public void run() {
         out.close();
         final long durationMillis = TimeUnit.MILLISECONDS.convert(System.nanoTime() - start, TimeUnit.NANOSECONDS);
 
-        LOG.info("[End] " + id);
+        LOG.info("[End  ] " + id);
```

Comment:
nitpick: not sure if you meant to include the two extra spaces in `[End  ]`

Reply:
I wanted to do that to line up with `[Start]`.